### DISCUSSION TITLE
Blank screen after updating to v1.7.44

### DISCUSSION THREAD

**Discussion by Xarmoda** [getgrav/grav#3788] [Help]
Hi, I was in the admin section of my GRAV powered site and I noticed there was a 'countdown' initiated for an upgrade from v1.7.43 to v1.7.44. I certainly don't remember enabling any automatic upgrades. Now both the administrative and public pages are completely blank. any tips on how to problem solve or fix this?

there is this in the cache:

<img width="609" alt="image" src="https://github.com/getgrav/grav/assets/111944369/3535e1f5-67e8-49e4-9e95-0b2af573aab1">

**Comment by rhukster** [comment]
The countdown is only to enable the upgrade button to be active.  It’s there to stop accidental pushing of the upgrade button. There are no automatic upgrades in Grav.  You have to push the upgrade button to initiate. 

That said if your site is blank it sounds like a php error.  You need to enable display of errors in php.ini or .htaccess or check your php error log.  They will tell you what is the root cause of the problem.

**Reply by Xarmoda** [reply]
Andy, I have been looking at the logs. There are a lot of errors with nothing easy to diagnose. Missing files etc. Something went wrong with the upgrade me thinks. I accept it's possible I clicked the upgrade option by accident.

It might be easier to reinstall over the top?  I'm running FreeBSD. It's not available as a FreeBSD port so I simply downloaded a zip file and extracted it.  The other two sites running GRAV (that didn't upgrade) are fine. I've done a full system upgrade since and an Apache restart. I'm confident it's not something to do with the wider system.
